Automattic/simplenote-macos
# Updated Tags List Design

Cleaning up the tags view with updated layout and icons. Also added the border back!

**To Test**
* Open the app, and view the tags drawer. Everything should still work including the drop down menu.
* Try switching themes, it should update properly. I made the tableView transparent with this PR, I think we don't need to set its bg color.

Screenshot:
<img width="401" alt="screen shot 2017-11-22 at 2 00 23 pm" src="https://user-images.githubusercontent.com/789137/33151824-889e670c-cf8e-11e7-9d1b-95bfc8f8fa10.png">
